std/sys/Http.hx

```diff
@@ -104,7 +104,7 @@ class Http extends haxe.http.HttpBase {
 				sock = new java.net.SslSocket();
 				#elseif python
 				sock = new python.net.SslSocket();
-				#elseif (!no_ssl && (hxssl || hl || cpp || (neko && !(macro || interp))))
+				#elseif (!no_ssl && (hxssl || hl || cpp || (neko && !(macro || interp) || eval)))
```

Comment:
This is becoming a very messy check. `pf_ssl`?

Reply:
I don't mind cleaning this up, but we can do that after merging this PR. I'll open a separate issue to track the fallout from this PR so we can merge it soonish.

Reply:
#9013